.repo-metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "client_documentation": "https://cloud.google.com/nodejs/docs/reference/appengine-admin/latest",
   "product_documentation": "https://cloud.google.com/appengine/docs/admin-api/",
-  "name": "appengine-admin",
+  "name": "appengine",
```

Comment:
It's confusing, because we were already using the `name` field for creating reference docs and the README. I believe we should actually leave this field `appengine-admin` (to keep these links constant).

And just update `api_shortname` (further down in the metadata file) to be `appengine`.